orbit/cmd/orbit/orbit.go

```diff
@@ -982,6 +982,22 @@ func main() {
 			}
 		}
 
+		// For macOS hosts, check if MDM enrollment profile is present and if it contains the
+		// custom end user email field. If so, report it to the server.
+		if runtime.GOOS == "darwin" {
```

Comment:
without much context, I wonder if we should do this in a loop like we do for the server URL/enroll secret.

going to approve as-is for now.